easybuild/scripts/bootstrap_eb.py

```diff
@@ -519,6 +532,8 @@ def main():
 # EasyBuild is a (set of) Python packages, so it depends on Python
 # usually, we want to use the system Python, so no actual Python dependency is listed
 allow_system_deps = [('Python', SYS_PYTHON_VERSION)]
+
+preinstallopts = '%(preinstallopts)s'
```

Comment:
where's the PR for the regualr EB easyconfigs? (and why are we still not using those?)